src/main/java/net/neoforged/neoforge/registries/RegistryBuilder.java

```diff
@@ -102,15 +102,19 @@ public RegistryBuilder<T> disableRegistrationCheck() {
      * @see NewRegistryEvent#register(Registry)
      */
     public Registry<T> create() {
+        return create(Lifecycle.stable(), false);
+    }
+
+    public Registry<T> create(Lifecycle lifecycle, boolean datapackRegistry) {
```

Comment:
There is no need to add this new parameter, just call `disableRegistrationCheck()`.

Reply:
I have added a new commit to resolve this